test/index.js

```diff
@@ -91,22 +45,24 @@ function response(spec, done) {
   };
 }
 
-test ('.run', () => assertResolved (run (middleware (42), 2), 42));
+test ('.run', () => equivalence (run (middleware (42), 2)) (resolve (42)));
 
-test ('.reject', () => assertRejected (evalState (null) (reject (42)), 42));
+test ('.reject', () => equivalence (evalState (null) (reject (42))) (rejectF (42)));
 
 test ('.fromComputation with a resolved future', () => {
   const actual = evalState (null) (fromComputation ((rej, res) => {
     res (42);
+    return () => null;
```

Comment:
I would say this is a breaking change. Fluture <12 wasn't complaining about missing cancellation functions.

Reply:
Ah, yes. That's a breaking change.